alibi_detect/cd/sklearn/tests/test_classifier_sklearn.py

```diff
@@ -0,0 +1,119 @@
+import pytest
```

Comment:
I think it would be great to test `_clone_model` method explicitly and separately with different kinds of estimators and user kwargs as there is quite a lot of logic in there. Would be useful to add in expected errors with `pytest.raises` and also test the outputs of the "custom" `predict_proba` methods.